tests/utils/test_utils.go

```diff
@@ -110,6 +138,161 @@ func ChangeWD(t *testing.T, newPath string) string {
 	return prevDir
 }
 
+func ReadCmdScanResults(t *testing.T, path string) *results.SecurityCommandResults {
+	content, err := os.ReadFile(path)
+	assert.NoError(t, err)
+	var cmdResults *results.SecurityCommandResults
+	if !assert.NoError(t, json.Unmarshal(content, &cmdResults)) {
```

Comment:
If the Unmarshal fails you return an empty struct that will allow to continue the test you are using this function in. This may cause us missing failing Unmarshals. I think we should return an error here so we can stop the test in case the file parsing has failed, or at least nil so we will get a nil dereference error to indicate the failure.

Reply:
what about this? why are we allowing failed unmarshal to continue?